AKO_GATEWAY_CHANGELOG.md

```diff
@@ -15,5 +15,35 @@ All notable changes to this project will be documented in this file. The format
 ### Added:
  - AKO now claims support for v1 for HttpRoute, Gateway, GatewayClass.
 
+## AKO-Gateway-1.13.1
+
+### Changed:
+ - AKO now accepts Gateway with some valid and some invalid listeners.
+ - AKO now allows creation/updation of HTTPRoute and Gateway in any order except for following cases which requires an explicit ako reboot to work.
+ 1. When one gateway attached to multiple routes, having all valid listeners, transitions to gateway with some invalid listeners.
+ 2. When one gateway attached to multiple routes, having all valid listeners, transitions to gateway with all invalid listeners.
+ 3. When one gateway attached to multiple routes, having some valid listeners, transitions to gateway with all invalid listeners.
+ 4. When one gateway attached to one route, having some valid listeners, transitions to gateway with all invalid listeners.
+ 5. When one gateway attached to one route, having all valid listeners, transitions to gateway with all invalid listeners.
+ - HTTPRoute deletion will not update Gateway listener status with "Virtual service deleted".
+ - If Secret specified in the TLS section does not exist, AKO will invalidate the Gateway listener and will not generate any configuration corresponding to it. If an existing Secret is deleted, then Parent VS configurations corresponding to it will get deleted.
+ - Hostname in Gateway listener is now an optional field.
+ - HTTPRoute -> spec -> rules -> matches is now an optional field.
+
+### Added:
+ - Support for Service of type NodePortLocal.
+ - Support for wildcard in Gateway->listener->Hostname.
+ - Support for wildcard prefixed Hostname in HTTPRoute.
+ - Support for ListenerConditionAccepted, ListenerConditionResolvedRefs, ListenerConditionProgrammed in Gateway Listener status.
```

Comment:
Can we club Conditions related changes on all objects in one sentence?

Reply:
Done